File: TP3_CPP/Codingan/External.cpp
#pragma once
#include <iostream>
#include <string>

using namespace std;

class External
{
private:
    string Koneksi;
    string MerkExternal;

public:
    External()
    {
    }

    External(string Koneksi, string MerkExternal)
    {
        this->Koneksi = Koneksi;
        this->MerkExternal = MerkExternal;
    }

    // Setter
    void setKoneksi(string Koneksi)
    {
        this->Koneksi = Koneksi;
    }

    void setMerkExternal(string MerkExternal)
    {
        this->MerkExternal = MerkExternal;
    }

    // Getter
    string getKoneksi()
    {
        return Koneksi;
    }

    string getMerkExternal()
    {
        return MerkExternal;
    }

    ~External()
    {
    }
};

File: TP3_Python/Codingan/External.py
class External:
    def __init__(self, koneksi="", merk_external=""):
        self.koneksi = koneksi
        self.merk_external = merk_external

    def set_koneksi(self, koneksi):
        self.koneksi = koneksi

    def set_merk_external(self, merk_external):
        self.merk_external = merk_external

    def get_koneksi(self):
        return self.koneksi

    def get_merk_external(self):
        return self.merk_external